From 690d70c64b4fc52627a01775c8d37d2e0076c0d5 Mon Sep 17 00:00:00 2001
From: Yinbo Zhu <yinbo.zhu@nxp.com>
Date: Tue, 25 Sep 2018 15:04:53 +0800
Subject: [PATCH 306/706] arm64: dts: ls1028a: Add dis_rxdet_inp3_quirk
 property to USB3 node

Add "dis_rxdet_inp3_quirk" boolean property to USB3 node. This property
is used to disable rx detection in P3 PHY mode.

Signed-off-by: Yinbo Zhu <yinbo.zhu@nxp.com>
(cherry picked from commit a8bb366bd7732bcf122653a2a48c397f7176b2cc)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 3f369e1dc133..2a38db19acbf 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -307,6 +307,7 @@
 			reg= <0x0 0x3100000 0x0 0x10000>;
 			interrupts= <0 80 0x4>;
 			dr_mode= "host";
+			snps,dis_rxdet_inp3_quirk;
 		};
 
 		usb1:usb3@3110000 {
@@ -314,6 +315,7 @@
 			reg= <0x0 0x3110000 0x0 0x10000>;
 			interrupts= <0 81 0x4>;
 			dr_mode= "host";
+			snps,dis_rxdet_inp3_quirk;
 		};
 
 		i2c0: i2c@2000000 {
-- 
2.17.1

